Dockerfile

```diff
@@ -37,7 +37,7 @@ RUN apk --no-cache add \
 RUN apk --no-cache add --virtual .build-deps \
      freetype-dev icu-dev zlib-dev libjpeg-turbo-dev libpng-dev libxml2-dev libxslt-dev libzip-dev linux-headers $PHPIZE_DEPS\
     && docker-php-ext-configure \
-         gd --with-freetype=/usr/include/ --with-jpeg=/usr/include/ \
+         gd ftp --with-freetype=/usr/include/ --with-jpeg=/usr/include/ \
```

Comment:
@px-benjamin see above: "ftp gd --with...". Maybe also remove the line break in 39+40.